x-pack/plugins/monitoring/server/kibana_monitoring/collectors/get_settings_collector.js

```diff
@@ -53,6 +53,14 @@ export async function checkForEmailValue(
   }
 }
 
+export function getEmailValueStructure(email) {
+  return {
+    xpack: {
+      default_admin_email: email
+    }
+  };
+}
```

Comment:
Since the collector constructor [spreads in any property](https://github.com/elastic/kibana/blob/master/src/server/usage/classes/collector.js#L44) given to it, we can make this function a part of the settings collector instance.

something like:
```
collectorSet.makeStatsCollector({	
  type: ...,
  async fetch: ...,
  getEmailValueStructure: email => ({
    xpack: { default_admin_email: email }
  })
})
```

In the line below where `getEmailValueStructure` is used in the fetch, it can use `this.getEmailValueStructure(defaultAdminEmail)`